manager/projects/templates/projects/_project_description.html

```diff
@@ -0,0 +1,23 @@
+{% load i18n stencila %}
+
+{% if project.description %}
+<p class="project-description">
+  {% with head=project.description|split_text %}
+  {# Important to keep the `.intro` and `.read-more` spans on one line without any white spaces to avoid HTML presentation issues  #}
```

Comment:
TODO: Replace this with [`spaceless` helper](https://docs.djangoproject.com/en/3.1/ref/templates/builtins/#spaceless)